optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -355,6 +385,28 @@ def __init__(
         # Set default parameters as best.
         self.best_params.update(DEFAULT_LIGHTGBM_PARAMETERS)
 
+        if study is None:
+            self.study = optuna.create_study(
+                direction="maximize" if self.higher_is_better() else "minimize"
+            )
+        else:
+            self.study = study
+
+        if self.higher_is_better():
+            if self.study.direction != optuna.structs.StudyDirection.MAXIMIZE:
+                metric_name = self.lgbm_params.get("metric", "binary_logloss")
+                raise ValueError(
+                    "Study direction is inconsistent with the metric {}. "
+                    "Please set 'maximize' to the direction.".format(metric_name)
```

Comment:
A nit.
```suggestion
                    "Please set 'maximize' as the direction.".format(metric_name)
```

Reply:
Thanks! I fixed it in https://github.com/optuna/optuna/pull/1032/commits/894a666c774e2b27f1811efad1e853c6add77cb3.

Reply:
Thanks!